core/src/main/java/io/questdb/Bootstrap.java

```diff
@@ -188,14 +188,20 @@ public CairoConfiguration getCairoConfiguration() {
             log.errorW().$(e).$();
             throw new BootstrapException(e);
         }
-
-        // metrics
         if (config.getMetricsConfiguration().isEnabled()) {
             metrics = Metrics.enabled();
         } else {
             metrics = Metrics.disabled();
             log.advisoryW().$("Metrics are disabled, health check endpoint will not consider unhandled errors").$();
         }
+        Unsafe.setWriterMemLimit(config.getCairoConfiguration().getWriterMemoryLimit());
```

Comment:
nit: would be nice to log the configured writer limit similar to how it's done for the RSS limit.